outbox-core/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+    `java-library`
+}
+
+dependencies {
```

Comment:
I see no use of commons-math nor guava.

Are `dependencies` added to show how to document them? Or you're sure they'll be used anyway?

Reply:
I have deleted them :)